sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/ListPathsOptions.java

```diff
@@ -89,11 +89,11 @@ public ListPathsOptions setRecursive(boolean recursive) {
      * The default value is false. Note that group and application Object IDs are not translated because they do not
      * have unique friendly names.
      *
-     * @param returnUpn The returnUpn value
+     * @param returnUserPrincipalName The returnUserPrincipalName value
      * @return the updated ListPathsOptions object
      */
-    public ListPathsOptions setReturnUpn(boolean returnUpn) {
-        this.returnUpn = returnUpn;
+    public ListPathsOptions setReturnUserPrincipalName(boolean returnUserPrincipalName) {
```

Comment:
I'm leaning towards just calling this `returnUserPrincipalName` as well. 

@JonathanGiles any concern around this? I know the boolean naming convention for fluent models is a bit more lax.

Reply:
We should prefix with the standard JavaBean prefixes. I would expect to see `is` and `set` used in here as appropriate. I don't know the proper nomenclature, but something like `isUserPrincipalNameReturned()` and `setUserPrincipalNameReturned()` might be better?

Reply:
Would it be best to change the name of the variable from returnUserPrincipalName to userPrincipalNameReturned then?